Source/Dafny.sln

```diff
@@ -28,236 +28,46 @@ Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "DafnyTestGeneration.Test",
 EndProject
 Global
 	GlobalSection(SolutionConfigurationPlatforms) = preSolution
-		Checked|.NET = Checked|.NET
-		Checked|Any CPU = Checked|Any CPU
```

Comment:
Don't we still need the Checked configuration, since that's what we actually build and package when releasing?

Reply:
If you look at how it's mapped to project configurations, you can see it's mapped to the Debug configuration of the projects, so it does the same as the solution level Debug configuration.

Reply:
The `package.py` script is at least attempting to use the `Checked` configuration: https://github.com/dafny-lang/dafny/blob/master/Scripts/package.py#L146

Although https://github.com/dafny-lang/dafny/issues/1390 makes me think that might not be working as intended, and I haven't gotten to the bottom of it yet.

Reply:
Ouch, I just checked and the `-c` option will accept any argument when a Project instead of a Solution is passed to `dotnet publish`. The default for `-c` is `Debug` so our `package.py` was making debug builds of Dafny. I'll update `package.py`.

Reply:
Ouch indeed!

It looks like the intention was to release `Debug` builds based on the way `Checked` is mapped in the solution file, or at least something other than `Release` builds. Let's figure out #1390 first before we remove all traces that something else was intended here.

Reply:
I've added the `CONTRACTS_FULL` constant to release configuration for DafnyDriver and DafnyPipeline and verified this turns on the contracts when running the RELEASE configuration.

Reply:
Okay, we've figured out #1390 and can make progress on that independently. I'll comment separately on the `package.py` change.

Reply:
Alright, we've figured out #1390 and can make progress on that independently.